app/models.py

```diff
@@ -1168,7 +1168,7 @@ class Brief(db.Model):
     clarification_questions = db.relationship(
         "BriefClarificationQuestion",
         order_by="BriefClarificationQuestion.published_at",
-        lazy='joined'
+        lazy='select',
```

Comment:
Yep, looking at sql statements produced there will just be one extra query to retrieve the clarification questions.